### PR TITLE
Updates Securing Software Repositories WG to Graduation stage

### PR DESCRIPTION
**Pull request by steiza**
Per https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md#submission-process

### CODE REVIEW

**Review by lehors — APPROVED** [review]
Thank you for getting us started on implementing our process!

File: README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ The following Technical Initiatives have been approved by the TAC:
 | Metrics & Metadata             | https://github.com/ossf/wg-metrics-and-metadata         | [Meeting Notes](https://docs.google.com/document/d/14_ILDhSK3ymKqUTQeQBRgJKgfiy_ePoGZIe8s7p3K5E/edit)  | Incubating |
 | Securing Critical Projects     | https://github.com/ossf/wg-securing-critical-projects   | [Meeting Notes](https://docs.google.com/document/d/1GFslP6elYCx27TUitdigDr1gsOItYkL0Vq7hTB9y4Lo/edit)  | Incubating |
 | Supply Chain Integrity         | https://github.com/ossf/wg-supply-chain-integrity       | [Meeting Notes](https://docs.google.com/document/d/1xPs2sSbH3I9Ich7OyLOzl85oJshnK8Q6WoAgREE5-zA/edit)  | Incubating |
-| Securing Software Repositories | https://github.com/ossf/wg-securing-software-repos      | [Meeting Notes](https://docs.google.com/document/d/1-f6m442MHg9hktrbcp-4sM9GbZC3HLTpZPpxMXjMCp4/edit)  | Incubating |
+| Securing Software Repositories | https://github.com/ossf/wg-securing-software-repos      | [Meeting Notes](https://docs.google.com/document/d/1-f6m442MHg9hktrbcp-4sM9GbZC3HLTpZPpxMXjMCp4/edit)  | Graduation |
```
Comment (marcelamelara) [inline README.md:61]: One super minor nit.

```suggestion
| Securing Software Repositories | https://github.com/ossf/wg-securing-software-repos      | [Meeting Notes](https://docs.google.com/document/d/1-f6m442MHg9hktrbcp-4sM9GbZC3HLTpZPpxMXjMCp4/edit)  | Graduated |
```

**Review by marcelamelara — APPROVED** [review]
Nice!! Thanks @steiza

**Review by SecurityCRob — APPROVED** [review]
<3

File: README.md (hunk 1 above)
Comment (steiza) [reply README.md:61]: This is fine by me! I wasn't sure what terminology to use here, the [template doc](https://github.com/ossf/tac/blob/main/process/templates/WG_NAME_graduation_stage.md) is name `graduation` but the [lifecycle stage](https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md#to-become-graduated) is of course `graduated`.

**Review by lehors — APPROVED** [review]
I'm approving again! :-)

**Review by marcelamelara — APPROVED** [review]

**Review by mlieberman85 — APPROVED** [review]
Lgtm

**Review by bobcallaway — APPROVED** [review]

**Comment by SecurityCRob** [comment]
I am pleased to congratulate our SCI WG in being our FIRST Graduated Working Group!  Yay you!!  Where your new badge with pride!  https://github.com/ossf/tac/blob/main/files/images/OpenSSF_StagesBadges_graduated.svg

**Pull request merged**